driver/random.hpp

```diff
@@ -1,32 +1,46 @@
 #ifndef GUARD_RANDOM_GEN_
 #define GUARD_RANDOM_GEN_
 
+#include <miopen/env.hpp>
+#include <iostream>
+#include <cstdlib>
 #include <random>
 
 std::minstd_rand& get_minstd_gen()
 {
-    static thread_local std::minstd_rand minstd_gen(std::random_device{}());
+    static thread_local std::minstd_rand minstd_gen{[]() {
+        auto external_seed = miopen::EnvvarValue("MIOPEN_DRIVER_PRNG_SEED", 100500);
```

Comment:
```suggestion
        static const auto external_seed = miopen::Value("MIOPEN_DRIVER_PRNG_SEED", 100500);
```
Actually it seems worth moving internal functions (like `miopen::EnvvarValue()` and `miopen::IsEnvvarValueEnabled()`) into `miopen::env::impl` namespace, and enlose the public stuff into `miopen::env` but I always do not have time for that ;)

Reply:
It is supposed to be inlined (probably I should add `inline` keyword). Moreover, it is already exposed to everyone, since it resides in a header.

Reply:
Meantime that construction
```
static thread_local std::minstd_rand minstd_gen {[]()[]{static const auto external_seed = miopen::Value()}()};
```
will trigger static member initialization 3 times just for no reasons. I insist to keep my version.

Reply:
First time, yes. Then it will not even call any function. But all right, the only thing I want to see here is `miopen::Value`, other stuff is up to you.